js/index.js

```diff
@@ -25,6 +25,10 @@ var ENDPOINTS = {
   }
 };
 
+function numberWithCommas(x) {
+  return x.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
Please leave a comment here for what this regex does, not immediately obvious at first glance.